samples/csharp/getting-started/Ranking_PersonalizedSort/PersonalizedRanking/Common/ConsoleHelper.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Microsoft.ML;
+using Microsoft.ML.Data;
+using PersonalizedRanking.DataStructures;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace PersonalizedRanking.Common
```

Comment:
Renaming:
I might recommend a title for this sample: "MSLR-WEB10K Ranking" or more simply, "Web Ranking".

I don't think there is [personalization](https://en.wikipedia.org/wiki/Personalized_search) in this dataset. Personalization in the ranking area generally means the each user gets individualized search results. In this dataset, there is not information about the user. This information would generally include { topics of interest of the user, demographics of the user, current location of the user, etc }.

Reply:
#resolved

Reply:
#resolved